sql-aggregates/profit-margin.sql

```diff
@@ -0,0 +1,18 @@
+WITH "profits" AS (
+  SELECT "title",
+          sum("amount") AS "totalPayments"
+    FROM "payments"
+    JOIN "rentals" USING ("rentalId")
+    JOIN "inventory" USING ("inventoryId")
+    JOIN "films" AS "f" USING ("filmId")
+    ORDER BY "totalPayments" desc
+    LIMIT 5
+)
+SELECT "title", "description"
+  FROM "films"
+  JOIN "inventory" USING ("filmId")
+  JOIN "rentals" USING ("inventoryId")
+  JOIN "payments" USING ("rentalId")
+  JOIN "profits" USING ("title")
+  ORDER BY "profits" desc
+  LIMIT 5;
```

Comment:
Great job attempting profit margin! You are on the right track using a CTE, but you actually need 2 CTEs. You need to get the total film revenue and the total film cost and find the difference to get the actual profit.